core/memory.go

```diff
@@ -112,24 +112,26 @@ func (memory *Memory) initMmap() {
 	}
 
 	//map sram
-	switch memory.romType {
-	case rom.LoROM:
-		for bankIndex := 0x70; bankIndex < 0x80; bankIndex++ {
-			for offset := 0; offset < 0x8; offset++ {
-				memory.mmap[bankIndex<<4|offset] = sramRegion
-				memory.mmap[(bankIndex+0x80)<<4|offset] = sramRegion
+	if len(memory.sram) > 0 {
```

Comment:
this is because it's not guaranteed that SRAMSize is not zero ?

Reply:
Yes some cartridges don't have any sram